src/Sulu/Bundle/AdminBundle/Resources/js/views/Datagrid/Datagrid.js

```diff
@@ -1,9 +1,9 @@
 // @flow
-import {action, observable} from 'mobx';
 import type {IObservableValue} from 'mobx'; // eslint-disable-line import/named
+import {action, observable} from 'mobx';
```

Comment:
Just out of curiosity: Have you changed the order everywhere like this, or did this you IDE?